janusgraph-cql/src/test/resources/docker/docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+# Copyright 2017 JanusGraph Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: '2'
```

Comment:
What's the purpose of this compose file? I don't see it mentioned or used anywhere.

Reply:
The compose is used to generate new config files for new cassandra versions.